src/rollup/types.d.ts

```diff
@@ -873,6 +875,7 @@ export interface NormalizedOutputOptions {
 	minifyInternalExports: boolean;
 	name: string | undefined;
 	noConflict: boolean;
+	onlyExplicitManualChunks: boolean | undefined;
```

Comment:
Unless `undefined` has a special meaning, I would normalize here to `boolean`. This is also what happens in `normalizeOutputOptions`.